backend/functions/lib/configs/authorization-configuration.test.ts

```diff
@@ -0,0 +1,58 @@
+describe('Authorization config tests', () => {
+  const originalEnv = process.env;
+
+  beforeAll(() => {
+    process.env = { ...process.env };
```

Comment:
What is the point of this? `thing = clone of thing with no changes`

Reply:
Is the idea to replace pointers to contents with pointers to different instances of the same contents?

Reply:
Because the tests mutate `process.env`.

Reply:
...and the copy leaves any starting env vars intact.